src/script/components/VerificationBadge/VerificationBadges.tsx

```diff
@@ -63,14 +71,77 @@ const title = (isMLSConversation = false): CSSProperties => ({
   marginRight: '4px',
 });
 
-export const VerificationBadges: React.FC<VerificationBadgesProps> = ({
+const useConversationVerificationState = (conversation: Conversation) => {
+  const {verification_state: proteusVerificationState, mlsVerificationState} = useKoSubscribableChildren(conversation, [
+    'verification_state',
+    'mlsVerificationState',
+  ]);
+  const mlsState = mlsVerificationState === ConversationVerificationState.VERIFIED ? MLSStatuses.VALID : undefined;
+  return {MLS: mlsState, proteus: proteusVerificationState};
+};
+
+export const UserVerificationBadges = ({user, groupId}: {user: User; groupId?: string}) => {
+  const {is_verified: isProteusVerified} = useKoSubscribableChildren(user, ['is_verified']);
+
+  return <VerificationBadges isProteusVerified={isProteusVerified} />;
+};
+
+export const DeviceVerificationBadges = ({
+  device,
+  userId,
+  groupId,
+  core = container.resolve(Core),
+}: {
+  device: ClientEntity;
+  userId: QualifiedId;
+  groupId?: string;
+  core?: Core;
+}) => {
+  const [isMLSVerified, setIsMLSVerified] = React.useState(false);
```

Comment:
Import useState directly from 'react'.

```suggestion
  const [isMLSVerified, setIsMLSVerified] = useState(false);
```

Reply:
Ah, I think this was generated by copilot, that's why I missed it. Good catch :)